core/templates/dev/head/domain/topic_viewer/ReadOnlyTopicObjectFactory.ts

```diff
@@ -0,0 +1,128 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
```

Comment:
Rename this file to read-only-topic-object.factory.ts and similarly change the test file name.